src/js/utils/helpers.js

```diff
@@ -48,10 +48,16 @@ export function each(collection, callback) {
         return;
     }
 
-    if (isArrayLike(collection) && !isObject(collection)) {
-        let l = collection.length,
-            i = 0;
-        for (; i < l; i++) {
+    if (isNodeList(collection)) {
```

Comment:
the current implementation uses a for loop to be able to use break, with your code that will not be possible.

Reply:
Revise coding style at https://github.com/biati-digital/glightbox/pull/165/commits/a8524327f6628d2451d9aeac9b844bce59b41993